zebra-state/src/service/check/tests/prop.rs

```diff
@@ -67,17 +74,32 @@ proptest! {
             let block1 = FinalizedBlock::from(Arc::new(block1));
             let commit_result = state.disk.commit_finalized_direct(block1.clone(), "test");
 
+            // the block was committed
             prop_assert_eq!(Some((Height(1), block1.hash)), state.best_tip());
             prop_assert!(commit_result.is_ok());
+
+            // the non-finalized state didn't change
             prop_assert!(state.mem.eq_internal_state(&previous_mem));
+
+            // the finalized state has the nullifiers
+            prop_assert!(state.disk.contains_sprout_nullifier(&expected_nullifiers[0]));
+            prop_assert!(state.disk.contains_sprout_nullifier(&expected_nullifiers[1]));
```

Comment:
Question: is `expected_nullifiers.len()` always `== 2`? Or should this be:

```suggestion
            for expected_nullifier in &expected_nullifiers {
                prop_assert!(state.disk.contains_sprout_nullifier(expected_nullifier));
            }
```

?

Reply:
It's always a fixed-length array with 2 items:
https://github.com/ZcashFoundation/zebra/blob/53779d2a3cc16a69fc449ede3f29da6063c1239c/zebra-chain/src/sprout/joinsplit.rs#L33-L34